docker-ml-dev/VERSION

```diff
@@ -0,0 +1 @@
+1.0
```

Comment:
We should adhere to the version we're on... e.g. 3.2 I think? Let's not "roll back" to 1.0.

Reply:
This is an old image. See comment below.